DllImportGenerator/DllImportGenerator/DllImportGenerator.cs

```diff
@@ -40,6 +40,9 @@ public void Execute(GeneratorExecutionContext context)
             var generatorDiagnostics = new GeneratorDiagnostics(context);
 
             var generatedDllImports = new StringBuilder();
+            generatedDllImports.AppendLine("#pragma warning disable CS8019");
```

Comment:
What's going on here?

Reply:
I added a default `using System;` to enable using the `.AsSpan()` extension method on arrays in the blittable array marshalling. This pragma disables the "unused using" warning when the namespace isn't used.

Reply:
Let's comment that. Thank you making the code clean. We need to make sure we document why we are disabling anything.

Reply:
We could just cast to manually create the span, but I like the `AsSpan` method because it's cleaner and because emitting a generic type in Roslyn is even more verbose than usual. I can change it to instead just create the span instance with the constructor.